### ISSUE TITLE
Support for pytest >= 7.0.0

### ISSUE THREAD

**Issue by cbradl01** [ckp95/pytest-parametrize-cases#6]
This package is not compatible with pytest >= 7.0.0. Would be great to have support for pytest >= 7.0.0. Sample error message below:

The conflict is caused by:
    The user requested pytest==7.0.1
    pytest-parametrize-cases 0.1.1 depends on pytest<7.0.0 and >=6.1.2

**Comment by ckp95** [comment]
I'll update it soon; for the time being you can just copy the contents of `case.py` into your project and import it and it should work.

By the way, how did you come across this package?

**Comment by cbradl01** [comment]
Excellent, thanks for the quick response! I'll try your advice tomorrow and come back to report if it worked.

I'm not sure exactly how I came across the package -- just stumbled upon it while searching for a better/clearer alternative for test parameterization. I think I found a post on stack overflow that led me here. 

Anyway, I really like the approach you have implemented, so thanks for the package!
